main.go

```diff
@@ -47,6 +47,7 @@ var (
 	wavefrontClient     *outputs.Client
 	fissionClient       *outputs.Client
 	grafanaClient       *outputs.Client
+	yandexS3Client      *outputs.Client
```

Comment:
```suggestion
	yandexClient      *outputs.Client
```

The client will be used for both S3 and Log